spec/lib/appsignal/integrations/mongo_ruby_driver_spec.rb

```diff
@@ -81,7 +81,7 @@
         transaction.should receive(:finish_event).with(
           'query.mongodb',
           'find | test | SUCCEEDED',
-          "{\"foo\":\"?\"}",
+          instance_of(Appsignal::Extension::Data),
```

Comment:
Are we not using a lot of useful testing here by only by only testing the type?

Can we create an instance of Appsignal::Extension::Data with these values instead?